### DISCUSSION TITLE
How does one configure multiple Tokens within catoken.properties?

### DISCUSSION THREAD

**Discussion by arshadnoor** [Keyfactor/ejbca-ce#106] [Q&A] [answered]
[This example](https://doc.primekey.com/ejbca/ejbca-integration/hardware-security-modules-hsm/utimaco-cryptoserver) in EJBCA 7.9.0.2 documentation shows one slot configured when using keys within that slot (but, unfortunately, still does not show up in Admin GUI - see [Discussion thread](https://github.com/Keyfactor/ejbca-ce/discussions/91#discussioncomment-3621672)).

Assuming there is a quick fix to the Admin GUI issue, how does one configure multiple slots without confusing EJBCA on which slot to use? For example, if I have 3 slots configured, how are the properties defined for specific Tokens within specific slots?

Thanks.

**Comment by primetomas** [comment, marked as answer]
You can configure as many different crypto tokens as you like in the UI, one for each slot. It will be obvious when you use the crypto token web UI.

**Reply by arshadnoor** [reply]
Since catoken.properties is **not** meant to be used with EJBCA v7, this can be closed. Thank you.
